### DISCUSSION TITLE
UTF-8 in SQLite

### DISCUSSION THREAD

**Discussion by ajmetz** [mojolicious/mojo#1821] [Q&A]
I read some docs, and figure I need....

use     DBD::SQLite::Constants qw(:dbd_sqlite_string_mode);

and then later...

{ sqlite_string_mode => DBD_SQLITE_STRING_MODE_UNICODE_FALLBACK }

Then I read Mojo::SQLite docs, and notice...
`Options for database handles, defaults to activating sqlite_unicode`

We are told in https://metacpan.org/pod/DBD::SQLite#Database-Handle-Attributes that we should
`Prefer sqlite_string_mode in all new code.`

So...can I pass an sqlite_string_mode via options ( https://metacpan.org/pod/Mojo::SQLite#options ) and that will override the deprecated sqlite_unicode ...?

**Comment by FGasper** [comment]
`sqlite_string_mode` is a new parameter in DBD::SQLite that improves upon (and obviates) `sqlite_unicode`. Mojo::SQLite probably just needs an update to use it.
